pkg/provider/kubernetes/crd/kubernetes.go

```diff
@@ -353,6 +353,26 @@ func (p *Provider) loadConfigurationFromCRD(ctx context.Context, client Client)
 			rootCAs = append(rootCAs, types.FileOrContent(caSecret))
```

Comment:
Could you please add a warning log when `RootCAsSecrets` is used?

Reply:
Done